game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -28,19 +31,30 @@ class GameThread(scoreThread: ScoreThread, teamIDs: List[Int]) extends Runnable
     states get gameId
   }
 
-  def createRankedGame = {
+  def createRankedGame: Game = {
+    actionMap.put(Game.RankedGameId, new HashMap())
     new Game(Game.RankedGameId, teamIDs)
   }
 
-  def run {
+  def run: Unit = {
     while (running) {
       updateGames
 
       Thread.sleep(MillisecondsPerTick)
     }
   }
 
-  def updateGames {
+  def transferActions: Unit = {
```

Comment:
Any reason not to call it before `updateGames` in `run`? :) Is it because it will come in a future PR?